nbgrader/exchange/releaseFeedback.py

```diff
@@ -0,0 +1,67 @@
+import os
```

Comment:
Maybe this file could be called `release_feedback.py`, to be consistent with the name of the nbgrader command?

Reply:
yes